src/main/java/com/github/fishio/MenuField.java

```diff
@@ -13,6 +13,8 @@
 public class MenuField extends PlayingField {
 	private GameThread gameThread;
 	
+	private static final int MAX_ENEMIES = 100;
+	
```

Comment:
Haha, good point. I completely forgot about that little setting there.
I'll change that back.

<performance